presentation-api/controlling-ua/reconnectToPresentation_success-manual.html

```diff
@@ -1,72 +1,49 @@
 <!DOCTYPE html>
 <meta charset="utf-8">
-<title>Presentation API, reconnect to presentation tests for Controlling User Agent (success - manual test)</title>
+<title>Reconnect to presentation success manual test</title>
 <link rel="author" title="Marius Wessel" href="http://www.fokus.fraunhofer.de">
-<link rel="help" href="http://w3c.github.io/presentation-api/#dfn-controlling-user-agent">
+<link rel="author" title="Louay Bassbouss" href="http://www.fokus.fraunhofer.de">
+<link rel="help" href="http://w3c.github.io/presentation-api/#dom-presentationrequest-reconnect">
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
 <script src="common.js"></script>
 
-<ol>
-    <li>Start the presentation with the blue button.</li>
-    <li>After the presentation ist running successfully, disconnect the connected device.</li>
-    <li>Reconnect the device and press the green button, to start the reconnection process.</li>
-</ol>
-
-<button id="startBtn" onclick="startPresentation()" style="background-color: #2aabd2;">Start Presentation</button>
-<button id="reconnectBtn" onclick="reconnectToPresentation()" style="background-color: #5cb85c;">Reconnect Presentation
-    Test
-</button>
+<p>Click the button below to start the manual test. Select a presentation device after the selection dialog is prompted. The test assumes that at least one presentation device is available. The test passes if a "PASS" result appears.</p>
+<button id="startBtn">Start Test</button>
 
 <script>
-    // disable the timeout function for the tests
-    // and call 'done()' when the tests cases are finished.
+    // disable timeout for manual tests
     setup({explicit_timeout: true});
-
-    // ----------
-    // DOM Object
-    // ----------
-    var startBtn = document.getElementById("startBtn"),
-            reconnectBtn = document.getElementById("reconnectBtn");
-
-    // ------------
-    // Request init
-    // ------------
-    var request = new PresentationRequest(presentationUrls);
-
-
-    // ----------------------------------------
-    // Helper Function - start the presentation
-    // ----------------------------------------
-    var startPresentation = function () {
-        return request.start()
-                .then(function (connection) {
-                    // save the presentation id within the
-                    // local storage of the browser
-                    localStorage["presId"] = connection.id;
+    async_test(function (t) {
+        var startBtn = document.getElementById("startBtn");
+        startBtn.onclick = t.step_func(function () {
```

Comment:
I'm the one who commented that the code was not wrapped in an `async_test` call but I think this goes too far. I would follow a "user clicks the button to start the test" approach and re-write as:

```js
startBtn.onclick = function () {
  startBtn.disabled = true;
  async_test(function (t) { ... });
};
```

(or use an `onclick` attribute that call a `startPresentation` function as in other tests)